presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -34,6 +34,7 @@
 import java.util.Locale;
 
 import static com.facebook.presto.operator.scalar.QuarterOfYearDateTimeField.QUARTER_OF_YEAR;
+import static com.facebook.presto.spi.StandardErrorCode.FUNCTION_NOT_FOUND;
```

Comment:
cmt msg ("Remove to_iso8601 function for new TIMESTAMP semantics"): `s/(complet)(ly)/\1e\2/g`